build/NuGetAdditionalFiles/RunCsc.cmd

```diff
@@ -1,2 +1,7 @@
-echo off
-dotnet %~dp0\csc.dll %*
+@echo off
+if defined DOTNET_HOST_PATH (
```

Comment:
No - I'm not sure if there's a significant difference. `if defined` looks stylistically cleaner, though, and I'd be less nervous about it (because of the following).

I was specifically researching behavior of unset variables - for example, my comment earlier in this thread:

>  when in a REPL, doing `echo "%an_unset_var%"` prints `"%an_unset_var%"`, but doing *the same thing* in a .cmd script prints `""`.

I was trying to validate correct behavior, since it seems like CMD vars don't have two distinct null/empty states, only the one - so I was trying to validate that the `set HOST_PATH=` line (which unsets the var `HOST_PATH`) does the right thing in all scenarios (so we don't execute literally `%HOST_PATH%dotnet`), but all I'm really sure about at this point is that it works when testing locally :(